### PR TITLE
Add logic to sections and updates

### PR DESCRIPTION
**Pull request by livfwds**
I may or may not be humming "We Are The Champions" as I submit this Pull Request.  That was *QUITE* the day Team.  Persistance (and Promises) prevail friends!

@ggrochow @inklin

### CODE REVIEW

File: lib/handlers/update.js [hunk 1]
```diff
@@ -31,73 +33,89 @@ var getSingleUpdate = function(req, reply) {
...
+    var s = section;
+    var title = s.title;
+    var intro = s.intro;
+    var body = s.body;
+    var imageUrl = s.imageUrl;
 
-    models.Update.create({
-      AccountId: currentUser.id
+    return models.Update.create({
+      AccountId: 1
```
Comment (ggrochow) [inline lib/handlers/update.js]: AccountId: 1 ?

File: lib/handlers/update.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 var models = require('../../models/index.js');
+var Boom   = require('boom');
+var Promise = require('bluebird');
 
 var getAccountUpdates = function(req, reply) {
```
Comment (ggrochow) [inline lib/handlers/update.js]: not a big deal ( just stylistic ) but this works the same
```
function getAccountUpdates(req, reply) { ... } 
```

File: lib/handlers/update.js (hunk 1 above)
Comment (livfwds) [inline lib/handlers/update.js]: danggit... I had to hard-code some things to get my curl commands to work.  will fix and resubmit

Comment (ggrochow) [inline lib/handlers/update.js]: you can just repush to this branch and it will be updated

File: lib/handlers/update.js [hunk 3]
```diff
@@ -31,73 +33,89 @@ var getSingleUpdate = function(req, reply) {
...
 }
 
-// TODO: will need to update this to update multiple sections
-var updateTheUpdate = function(req, reply) {
-  var title = req.payload.title;
-  var intro = req.payload.intro;
-  var body = req.payload.body;
-  var imageUrl = req.payload.imageUrl;
-  var UpdateId = req.params.id;
+var updateUpdateSections = function(req, reply) {
```
Comment (ggrochow) [inline lib/handlers/update.js]: same thing with 
```
function updateUpdateSections(req, reply) { ... }
```
but i mean im like a semicolon not user so what do I know about stylistic things

File: lib/handlers/update.js (hunk 2 above)
Comment (livfwds) [inline lib/handlers/update.js]: was just trying to keep it consistent with what we have everywhere else :)

Comment (inklin) [inline lib/handlers/update.js]: Yep let's keep it with the var since that's how it is in all the other files :)

File: lib/handlers/update.js [hunk 4]
```diff
@@ -15,9 +17,9 @@ var getAccountUpdates = function(req, reply) {
...
 
 var getSingleUpdate = function(req, reply) {
-  var UpdateId = req.params.id;
+  var update = req.payload.update;
 
-  models.Update.findById(UpdateId).then(function(update){
+  models.Update.findById(update.id).then(function(update){
     update.getSections().then(function(sections){
       reply({
         statusCode: 200,
```
Comment (inklin) [inline lib/handlers/update.js]: Is there data being returned in this reply?

File: lib/handlers/update.js (hunk 3 above)
Comment (inklin) [inline lib/handlers/update.js]: haha you're just more confident about when you need semicolons and when auto semi colon insertion works ;)

Comment (ggrochow) [inline lib/handlers/update.js]: I just forget semicolons and you don't notice it when I don't have them in the first place

File: lib/handlers/update.js [hunk 5]
```diff
@@ -15,9 +17,9 @@ var getAccountUpdates = function(req, reply) {
 }
 
 var getSingleUpdate = function(req, reply) {
-  var UpdateId = req.params.id;
+  var update = req.payload.update;
 
-  models.Update.findById(UpdateId).then(function(update){
+  models.Update.findById(update.id).then(function(update){
```
Comment (inklin) [inline lib/handlers/update.js]: We can add an error function here for "Failed to get sections for Update" :)

File: lib/handlers/update.js (hunk 3 above)
Comment (inklin) [inline lib/handlers/update.js]: haha! it's all good.

this pr looks good, i'm going to merge it in.

**Pull request merged**